content/en/docs/concepts/storage/volumes.md

```diff
@@ -543,6 +543,68 @@ spec:
       type: FileOrCreate
 ```
 
+### image
+
+{{< feature-state feature_gate_name="ImageVolume" >}}
+
+An `image` volume source represents an OCI object (a container image or
+artifact) pulled and mounted on the kubelet's host machine.
```

Comment:
BTW, we don't know how the container runtime achieves this. Not all runtimes will need to do the mount on the host.

Reply:
I changed the phrasing to relax the mount restriction.